scripts/expo-upload-sourcemaps.js

```diff
@@ -147,6 +149,13 @@ if (!sentryOrg || !sentryProject) {
     console.log(`${SENTRY_PROJECT} resolved to ${sentryProject} from expo config.`);
   }
 }
+if (pluginConfig.url) {
+  sentryUrl = pluginConfig.url;
+} else {
+  sentryUrl
+}
+console.log(`${SENTRY_PROJECT} resolved to ${sentryProject} from expo config.`);
+
```

Comment:
Although, I agree with loading the URL from the Expo Plugin Configuration.

The current code won't work, as `pluginConfig` variable is not in the scope.

This should be under `if (!sentryOrg || !sentryProject) {`, which needs to be updated to include `sentryUrl`.